### PR TITLE
Feature/5854 differenciate link by protocol

### PR DESCRIPTION
**Pull request by HavierD**
don't want WMS or WFS to show on frontend

### CODE REVIEW

File: indexer/src/main/java/au/org/aodn/esindexer/utils/LinkUtils.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+/**
+ * Protocols here are referenced from old portal project(Grails). Currently, didn't
+ * implement all of them. Others can be added if needed.
+ */
+@SuppressWarnings("unused")
+public class LinkUtils {
+
+    // protocols
+    private final static List<String> WMS = Arrays.asList(
```
Comment (utas-raymondng) [inline indexer/src/main/java/au/org/aodn/esindexer/utils/LinkUtils.java:14]: Will regex match better to avoid diff version?

**Review by utas-raymondng — CHANGES_REQUESTED** [review]
minor comment

File: indexer/src/main/java/au/org/aodn/esindexer/utils/LinkUtils.java (hunk 1 above)
Comment (HavierD) [reply indexer/src/main/java/au/org/aodn/esindexer/utils/LinkUtils.java:14]: Thanks. I will check with Nat to know whether new WMS & WFS will add in the future, and if so, what kind of regex i should use.

Comment (utas-raymondng) [reply indexer/src/main/java/au/org/aodn/esindexer/utils/LinkUtils.java:14]: I doubt she know but from the link WWW:DOWNLOAD-1.0-http--download the 1.0 is version and the other similar x.y.z

**Review by utas-raymondng — APPROVED** [review]
LGTM

**Pull request merged**